packages/core/src/http-client/http-client.ts

```diff
@@ -30,6 +35,13 @@ export async function buildHttpRequest(
   destination: Destination | DestinationNameAndJwt,
   customHeaders?: Record<string, any>
 ): Promise<DestinationHttpRequestConfig> {
+  if (customHeaders) {
+    logger.warn(
+      `The custom headers are provided with the keys: ${Object.keys(
```

Comment:
Perhaps also add a sentence what this means: `These keys will overwrite the headers created by the SDK`.